src/module/Ui/templates/entity/view/partials/input-challenge.twig

```diff
@@ -0,0 +1,21 @@
+{% set revision = entity.getCurrentRevision() %}
```

Comment:
not sure if this is required, because this template is only used by one view. Maybe merge `partials/input-challenge.twig` with `view/input-string-normalized-match-challenge.twig`